trail/gallery/tests.py

```diff
@@ -1,3 +1,21 @@
+# Not sure how to get this working, may look at a seperate time
+#
 # from django.test import TestCase
-
-# Create your tests here.
+# from gallery.views import get_images
+# from uploads.models import Thumbnails
+#
+#
+# class GalleryImageTestCase(TestCase):
+#     def setUp(self):
+#         objlist = []
+#         for i in range(100):
+#             objlist.append(Thumbnails({"wcs-id": i, "small": "", "large": ""}))
+#         Thumbnails.objects.create(objlist)
```

Comment:
Try:

```
from django.test import TestCase
from gallery.views import get_images

from django.apps import apps

UploadInfo =  apps.get_model('upload', 'UploadInfo')
Metadata = apps.get_model('upload', 'Metadata')
Thumbnails = apps.get_model('upload', 'Thumbnails')
Wcs = apps.get_model('upload', 'Wcs')

class GalleryImageTestCase(TestCase):
    def setUp(self):
        info = UploadInfo()
        info.save()

        meta = Metadata(
            upload_info=info,
            processor_name="",
            standardizer_name="",
            instrument="",
            telescope="",
            science_program="",
            obs_lon=1.0,
            obs_lat=1.0,
            obs_height=1.0,
            datetime_begin='2009-10-17T01:53:42.1',
            datetime_end='2009-10-18T01:53:42.1',
            exposure_duration=30.0,
            filter_name='i'
        )
        meta.save()

        thumbs = []
        for i in range(100):
            wcs = Wcs(
                metadata = meta,
                wcs_radius=1.0,
                wcs_center_x=1.0,
                wcs_center_y=1.0,
                wcs_center_z=1.0,
                wcs_corner_x=1.0,
                wcs_corner_y=1.0,
                wcs_corner_z=1.0
                )
            wcs.save()
            thumbs.append(Thumbnails(wcs=wcs, small="", large=""))

        Thumbnails.objects.bulk_create(thumbs)

    def testImageListSame(self):
        """test that the pages are working"""
        print(Thumbnails.objects.all().values("wcs"))
```

Unfortunately all relationships on the table have to be respected, even in testing, so without creating the dependencies for the tables to reference you will not be able to insert the `Thumbnails` (depend on `Wcs`, depend on `Metadata`, depends on `UploadInfo`).

Open an issue to remind me to punt this into a function that we can have somewhere in global `test_utils.py`for convenience. Re: the comments in your last commit - you can run tests with `python manage.py test -v 2` for pretty line-by-line output. If you add docstrings to your test class - that's the text that will appear in that output with some dots and an `OK` or a `FAIL` message.